packages/expo-modules-core/src/fetch/FetchResponse.ts

```diff
@@ -0,0 +1,111 @@
+import { ReadableStream } from 'web-streams-polyfill';
```

Comment:
Do you think we should eventually implement streams too?

Reply:
web-streams-polyfill looks good for maintenance. would consider if we don’t want to pull the package. it also takes some effort to implement the stream classes.